code/_onclick/item_attack.dm

```diff
@@ -1,4 +1,21 @@
 
+/obj/item/proc/melee_attack_chain(mob/user, atom/target, params)
+	if(!tool_attack_chain(user, target) && pre_attack(target, user, params))
```

Comment:
Do an early return here.

Reply:
I cant this is required to be processed to go to afterattack() now

Reply:
Explained what an early return without altering functionality is through discord.